src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -4076,3 +4099,25 @@ def aks_pod_identity_exception_update(cmd, client, resource_group_name, cluster_
 def aks_pod_identity_exception_list(cmd, client, resource_group_name, cluster_name):
     instance = client.get(resource_group_name, cluster_name)
     return _remove_nulls([instance])[0]
+
+
+def _ensure_cluster_identity_permission_on_kubelet_identity(cli_ctx, cluster_identity_object_id, scope):
+    managed_identity_operator_role = 'Managed Identity Operator'
```

Comment:
define as consts?

Reply:
Done. Thanks for reviewing and reminding!